React/CxxBridge/RCTCxxBridge.mm

```diff
@@ -157,6 +158,8 @@ static void registerPerformanceLoggerHooks(RCTPerformanceLogger *performanceLogg
   };
 }
 
+static BOOL runtimeIsInitialized;
```

Comment:
It concerns me that this is a `static` because we can (and do sometimes) have multiple bridges that get loaded and unloaded during the lifetime of an app.

Reply:
Yea I get that concern. I had to make it static because we'd have multiple bridge instances in one feature go through this so once the global runtime for that feature was done, all the bridges had to know. I did make some other changes in the meantime though so will test this again without it.

I suppose the downside to it being static is just that we're just back to where we were before this change where we still crash in that race condition.